traefik/values.yaml

```diff
@@ -14,6 +14,8 @@ deployment:
   annotations: {}
   # Additional pod annotations (e.g. for mesh injection or prometheus scraping)
   podAnnotations: {}
+  # Additional containers (e.g. for metric offloading sidecars)
+  additionalContainers: []
```

Comment:
An example would help the end user.

Reply:
And you are breaking with this change. You can't bump a minor version

Reply:
populating this as a tables would have never generated a sucessful install, only `maps` would be sucessful. as such, this may qualify as a patch.